website/content/docs/concepts/events.mdx

```diff
@@ -0,0 +1,150 @@
+---
+layout: docs
+page_title: Events
+description: >-
+  Events are an experimental feature that allows Vault and plugins to exchange arbitrary activity data
+  within Vault and with external subscribers via WebSockets.
+---
+
+# Events
+
+-> **Important:** Events are experimental, and may have backwards incompatible changes introduced in subsequent versions of Vault, or be removed altogether.
+
+Events are arbitrary, **non-secret** data that can be exchanged between producers (Vault and plugins)
+and subscribers (Vault components and external users via the API).
+Events are delivered on a *best-effort* basis, so do not have delivery guarantees at this time.
+
+As of Vault 1.13, events are not considered production-ready, and so are disabled by default when starting a Vault server.
+Events can be enabled via the `events.alpha1` [experiment flag](/vault/docs/configuration#experiments) in the Vault
+configuration or on the command-line:
+
+```shell-session
+$ vault server -experiment events.alpha1
+```
+
+~> Note: Experiments can only be enabled at startup, and cannot be disabled or enabled at runtime.
+
+## Event Types
+
+<!-- This information will probably be migrated to the plugin pages eventually -->
```

Comment:
I figured this would probably be a bigger discussion as the events expand to more plugins, as there isn't a clear right answer. But it's super useful to have these all in one place for now.

Idea: we could add the events in the API docs with a special directive that also adds them to a table here, like a TOC.

Reply:
Oh yeah sorry I didn't mean in this PR. What you've already got is definitely the best first step. That idea would be really nice. In a similar vein of thought, I was thinking it would be nice if plugins could self-report their own registry of event types - it feels like we could build some nice features on top of that in the future.